config/route53/config.go

```diff
@@ -0,0 +1,124 @@
+/*
+Copyright 2021 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package route53
+
+import (
+	"strings"
+
+	"github.com/crossplane-contrib/terrajet/pkg/config"
+)
+
+// Configure route53 resources.
+func Configure(p *config.Provider) {
+	p.AddResourceConfigurator("aws_route53_delegation_set", func(r *config.Resource) {
+		r.ExternalName = config.IdentifierFromProvider
+	})
+	p.AddResourceConfigurator("aws_route53_health_check", func(r *config.Resource) {
+		r.ExternalName = config.IdentifierFromProvider
+	})
+	p.AddResourceConfigurator("aws_route53_hosted_zone_dnssec", func(r *config.Resource) {
+		r.ExternalName = config.IdentifierFromProvider
+		r.References = config.References{
+			"hosted_zone_id": config.Reference{
+				Type: "Zone",
+			},
+		}
+	})
+	p.AddResourceConfigurator("aws_route53_key_signing_key", func(r *config.Resource) {
+		r.ExternalName = config.IdentifierFromProvider
+		r.References = config.References{
+			"hosted_zone_id": config.Reference{
+				Type: "Zone",
+			},
+			"key_management_service_arn": config.Reference{
+				Type: "github.com/crossplane-contrib/provider-jet-aws/apis/kms/v1alpha1.Key",
```

Comment:
Is the default extractor okay in this case?

Reply:
It wasn't. Just added an ARN extractor for KMS key https://github.com/crossplane-contrib/provider-jet-aws/pull/130/commits/ee989730ca0eea9cd7e483ed017edcd03b39fc24